datasets/wine_quality_red/metadata.yaml

```diff
@@ -17,22 +17,22 @@ target:
     Wine quality score based on sensory data, median of at least 3 evaluations made by wine experts.
   code: ranges from 0 (very bad) to 10 (very excellent)
 features: # list of features in the dataset
-  - name: fixed_acidity
+  - name: fixed acidity
```

Comment:
Is this an oversight or did we rename the feature names here?

Reply:
I changed the feature names based on [original dataset in pmlb1.0](https://github.com/EpistasisLab/penn-ml-benchmarks/blob/master/datasets/classification/wine-quality-red/README.md) . We may change them later if the dataset file ending in tsv.gz was updated based on [openml](https://www.openml.org/u/869). I think using "_" in feature name should be OK.

Reply:
Ah I see that you reverted the dataset back. Yup sounds good!